src/client/_clientwindow.py

```diff
@@ -1152,7 +1152,8 @@ def handle_invalid(self, message):
         raise Exception(message)
 
     def handle_stats(self, message):
-        self.statsInfo.emit(message)
+        # self.statsInfo.emit(message)  # currently useless -> safe some time
```

Comment:
it is significant .. loading the many thousand users for no purpose (the league is 'out of order'), when you want to have a look at ladder ratings.